src/components/leaderBoard.js

```diff
@@ -97,10 +97,19 @@ class LeaderBoard extends Component {
 
     render() {
         return  <div>
-            <h2>LeaderBoard</h2>
-            <ListGroup>
+            <table class="table table-striped">
+              <thead>
+                <tr>
+                  <th scope="col">Rank</th>
+                  <th scope="col">Hero</th>
```

Comment:
Maybe rename to "Player"? Hero seems over the top. I'm fine with it if you want to keep it though.

Reply:
sure, I'll rename